test/tracing/frcnn/test_frcnn_tracing.cpp

```diff
@@ -0,0 +1,25 @@
+#include <torch/torch.h>
```

Comment:
Interesting, so we don't need to include anything for torchvision and it already shows up as expected?